### PR TITLE
Subscription pack PayPal Checkout gets "Error: Access Denied"

### PR DESCRIPTION
**Pull request by jayyx2**
When you click "Proceed" the following error is seen from Paypal:
Access Denied.

You don't have permission to access "/webscr/?cmd=_xclick-subscriptions&business=info%40thedreadmachine.com&a3=50&mc_amount3=50&p3=1&t3=Y&item_name=Acolyte&custom=%7B%22billing_amount%22%3A50%2C%22type%22%3A%22pack%22%2C%22user_id%22%3A26456%2C%22coupon_id%22%3A%22%22%2C%22subtotal%22%3A%2250%22%2C%22tax%22%3A0%7D&shipping=0&no_note=1¤cy_code=USD&item_number=12449&rm=2&return=https%3A%2F%2Fwww.thedreadmachinetest.com%2Fsubscription%2F%3Faction%3Dwpuf_paypal_success&cancel_return=https%3A%2F%2Fwww.thedreadmachinetest.com%2Fsubscription%2F%3Faction%3Dwpuf_paypal_success¬ify_url=https%3A%2F%2Fwww.thedreadmachinetest.com%2F%3Faction%3Dwpuf_paypal_success&src=1&sra=1&srt=0&recurring=1" on this server.
Reference #d8a0e489-ea89-45e6-bf9a-f7f0c0d9373f
Timestamp: 1601643396

Troubleshooting of this issue shows that removing the trailing / from the $return_url fixes the issue. Adding "untrailingslashit" to this removes the trailing slash from the get_permalink call.

### CODE REVIEW

**Comment by tareq1988** [comment]
@FaraziF @shabnam611 can you test and reproduce the issue?

**Comment by FaraziF** [comment]
In the following scenario, can't reproduce this type of permission related issue in our latest develop branch and latest released version 3.5.2.

**Comment by jayyx2** [comment]
I have a test site you can use if necessary. I am able to easily reproduce this on both my test and production site.

**Comment by jayyx2** [comment]
> In the following scenario, can't reproduce this type of permission related issue in our latest develop branch and latest released version 3.5.2.

This is not a permissions issue, it's a Paypal API issue. It seems the Paypal API you're using doesn't allow you to pass trailing slashes in certain scenarios.

**Comment by FaraziF** [comment]
@jayyx2 Sorry to say that I mistake to reproduce this issue. This issue exists in our product. Hopefully, quickly we will test your pull request & merge it in our develop branch. Thank you for contributing to our product.

**Comment by FaraziF** [comment]
I've tested and can confirm this pull request fixes the issue.

File: lib/gateway/paypal.php [hunk 1]
```diff
@@ -194,7 +194,6 @@ public function prepare_to_send( $data ) {
                 'currency_code' => $data['currency'],
                 'item_number'   => $data['item_number'],
                 'rm'            => 2,
-                'return'        => $return_url,
```
Comment (jayyx2) [inline lib/gateway/paypal.php:197]: actually, this line should stay...

Comment (tareq1988) [reply lib/gateway/paypal.php:197]: Yes, please revert this line back.

**Pull request merged**